.circleci/config.yml

```diff
@@ -1,4 +1,4 @@
-version: 2
+version: 2.1
```

Comment:
This stays `2`. It's not our version, it's the version of circle ci.

Reply:
yeah, "Enable build processing (preview)

Enable support for new config 2.1 features along with API build triggering and auto-cancellation of redundant workflows."

Reply:
Ok, sounds dangerous. Let's see what happens. Mostly just need to watch deploys to App Store and Google Play. Not sure what happens if you cancel in the middle of submitting and how that will effect our build pipeline.

Reply:
Doesn't happen on Master, all commits are built on master